src/applications/claims-status/sass/claims-status.scss

```diff
@@ -1023,6 +1023,10 @@ h1:focus {
   }
 }
 
+.appeals-next-list-no-separator li.next-event {
+  margin: 1.5em 0 2em;
```

Comment:
we have some margin [utilities](https://department-of-veterans-affairs.github.io/vets-design-system-documentation/utilities/margins.html) that are available now.